src/state/slices/opportunitiesSlice/utils.ts

```diff
@@ -0,0 +1,29 @@
+import type { AccountId } from '@shapeshiftoss/caip'
+
+import type { StakingId, UserStakingId } from './opportunitiesSlice'
+
+type UserStakingIdParts = [accountId: AccountId, stakingId: StakingId]
+
+export const deserializeUserStakingId = (userStakingId: UserStakingId): UserStakingIdParts => {
+  const parts = userStakingId.split('*')
```

Comment:
pull this out to `STAKING_ID_DELIMITER` so we can change it in once place in future if necessary, i've burnt myself doing this before

Reply:
5e0370883b6f94625b75bd8c84cfcf35358725fa